### PR TITLE
Ignore vendor/ directory and simplify build step

### PR DESCRIPTION
**Pull request by phillipj**
Just a couple of trivial housekeeping chores to makes life a little easier when working on this project.

Primarily to make the build from source setup more convenient and be in line with what's described in [README.md](README.md).

/cc @mahpatil

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -10,7 +10,7 @@ get: clean
 	@echo "Get..."
 	@GOPATH=$(GOPATH) go get github.com/hashicorp/terraform/plugin
 
-build: clean
+build: get
```
Comment (phillipj) [inline Makefile:13]: `clean` will still be run implicitly via `get`

**Comment by tazjin** [comment]
While at it you can also remove the `.frm` file, that's no longer relevant.

**Comment by phillipj** [comment]
Yeah I've been struggling quite a bit trying to build with Repeatr using that formula.. While on that subject, got any thoughts on what would be the tools to use for consistent builds nowadays?

I've been thinking:
- godeps for dependencies
- docker for enforcing the golang version

But I'm not that updated in the golang sphere, so there might be better alternatives out there 🤷‍♂️

**Comment by tazjin** [comment]
> Yeah I've been struggling quite a bit trying to build with Repeatr using that formula

Probably best not to bother ;-)

> While on that subject, got any thoughts on what would be the tools to use for consistent builds nowadays?

On my primary other Go project I use [Nix](https://nixos.org/), which I use for basically everything work and private these days. It makes it quite easy to pin dependencies & build repeatably.

From kontemplate, check out [this file](https://github.com/tazjin/kontemplate/blob/master/default.nix) which builds and runs tests. It's also configured to run `go vet` in the check phase.

Dependencies are pinned [here](https://github.com/tazjin/kontemplate/blob/master/deps.nix). Last but not least (this is probably less relevant to this project), [release.nix](https://github.com/tazjin/kontemplate/blob/master/release.nix) builds release binaries for different platforms.

Nix has the additional advantage that it won't just pin the Go compiler version, but the entire universe. Running a Nix build a few years later will still work fine.

Nix will work fine on Linux & OS X, but it probably won't be an option if you have Windowsers working on this project.

**Comment by phillipj** [comment]
Oho thanks a lot for the thorough explanation using Nix! Absolutely something I'll give a shot using.

In the meantime I'll do as you suggested, delete Repeatr related files and references in the README.

**Comment by phillipj** [comment]
@mahpatil you've got any thoughts and/or preferences on reproducible builds by any chance?

**Comment by mahpatil** [comment]
no real preference, haven't used Nix but worth giving it a shot.

**Review by mahpatil — APPROVED** [review]

**Comment by phillipj** [comment]
Thanks for a quick response @mahpatil!

I'll merge this and create a follow up issue getting reproducible builds in place. Would be nice to push a new release w/binaries after getting that in place.

**Pull request merged**